.github/workflows/doc-build.yml

```diff
@@ -162,7 +162,7 @@ jobs:
           EOF
           echo '</head>' >> ./docs/CHANGES.html
           echo '<body>' >> ./docs/CHANGES.html
-          (cd /sage/local/share/doc/sage/html && git diff HEAD^ -- *.html; rm -rf .git) > ./docs/diff.txt
```

Comment:
The purpose of removing the .git directory is that it does not end up in `docs.zip`

Reply:
Sure. I was mindless...